src/wrappers/organizations.ts

```diff
@@ -76,4 +76,13 @@ export default class {
 
     return data;
   }
+
+  public async addMemberToOrg(
```

Comment:
this should be just `addMember` the org is implied because it is on the organizations wrapper

Reply:
Got it! Thanks :)